Expand YMMB Procedures (#385)
vatpac-technology/sops: docs/aerodromes/classd/Moorabbin.md

Refer to [Class D Tower Separation Standards](../../../separation-standards/classd) for more information.

## Runway Modes

Preferred runway modes are as follows (in order of preference):

1. Runway 35L/35R  
2. Runway 17L/17R  
3. Runways 13/31 (equal preference)

> Runway 04/22 may only be used operationally. Consideration must be given to it's limited length.

Author: Dunno, that's pre-existing text that was there in the old version :)

Usually in real life I know they tend to never use 22 in practicality.

> Runway 04/22 may only be used operationally. Consideration must be given to it's limited length.

Author: Any preference on the wording here. They have never used 22 in my couple years of going in and out. The runway now has warehouses on one side. They tend to favour cross winds for 13/17 etc over ever using this runway. 

17/35 is preferred since it lines up with the tower. Easier to see out which is why I listed it as preferred

> Runway 04/22 may only be used operationally. Consideration must be given to it's limited length.

Author: Just had a discussion with a real world MB controller

He has indicated 
> It can't be nominated, only used on request/requirement 
> Even when xw is over tolerance for all other rwys, i believe 22/04 will still not be nominated
> I've been at MB almost a year, seen it used once on request

I asked what would be a example of a requirement or request:
> Requirement would be emergency/pan, request is basically just shits and gigs
> Outside no traffic random requests, it's only used in emergencies
> Request would only be granted if no traffic
> Runway 04 likely not be used due to vacating on a runway but likely be ok vacate B on request but no-one ever asks for 04 usually

Maybe something like:

Runway 04/22 is not nominated by default and is only available on request or when operationally required for emergencies. Requests for Runway 22 may be granted if traffic conditions allow. Runway 04 is less likely to be approved due the need to vacate onto an intersecting runway.

| BAW         | Join base 13R  | Join base 17R | Join oblique downwind 31L | Join base 35L |
| CARR        | Join downwind 13R | Join downwind 17R | Join oblique base 31L | Join final 35L |
| GMH         | Join downwind 13L | Join base 17L | Join oblique base 31R | Join base 35R |
| ACE         | Join base 13L | Join oblique base 17L | Join base 31R | Join oblique downwind 35R |
| SHOAL       | Join base 13L | Join base 17L | Join base 31R | Join base 35R |

CTR Entry Altitude: `A010`.

## Circuits

### Fixed wing

Circuit altitude: `A010`.

| Runway  | Day Circuit Direction | Night Circuit Direction |
|---------|------------------------|--------------------------|
| 13L     | Left                  | Left                    |
| 13R     | Right                 | N/A                     |
| 31L     | Left                  | N/A                     |
| 31R     | Right                 | Right                   |
| 17L     | Left                  | Left                    |
| 17R     | Right                 | N/A                     |
| 35L     | Left                  | N/A                     |
| 35R     | Right                 | Right                   |

> ## Helicopters

Author: Worth noting that the helicopter operations are actually very similar to almost all Class D airports I've flown to in helicopters. So that isn't specific to per sae with Moorabbin. At least with the radio phraseology of calling airborne etc.

> ## Helicopters

Reviewer: > Worth noting that the helicopter operations are actually very similar to almost all Class D airports I've flown to in helicopters. So that isn't specific to per sae with Moorabbin. At least with the radio phraseology of calling airborne etc.

Absolutely taking no issue with how close to RL operations what you've done up is, it's all great. Unfortunately we do have to make some concessions from RL operations, and this does end up affecting Metro D's more than most, as they are the most vulnerable to having more local variations to procedures, which isn't suitable for VATSIM at times.

I apologise if I've requested changes on anything that's not your work, just trying to get it in to a state where we're all happy.

|---------|------------------------|--------------------------|
| 13L     | Left                  | Left                    |
| 13R     | Right                 | N/A                     |
| 31L     | Left                  | N/A                     |
| 31R     | Right                 | Right                   |
| 17L     | Left                  | Left                    |
| 17R     | Right                 | N/A                     |
| 35L     | Left                  | N/A                     |
| 35R     | Right                 | Right                   |

## Helicopters

### Circuits

#### Day VFR Circuits
Helicopter circuits are conducted from designated areas based on the active runway:

- **RWY 17/35**: From the **Eastern Grass**, which encompasses all areas east of runway 35R.
- **RWY 13/31**: From the **Western Triangle**, which extends:
    - From TWY A to the boundary fence,
    - Up to the centerline of TWY A4,
    - Then across to the edge of TWY B and runs parallel to TWY B to the boundary fence.

<figure markdown>
![Eastern Grass and Western Triangle](img/mbhelicircuitareas.png){ width="700" }
<figcaption>Eastern Grass and Western Triangle</figcaption>
</figure>

> Circuit operations are conducted parallel to the duty runway and inside the fixed-wing circuit.

Reviewer 2: What is the helicopter circuit altitude? I'm assuming A007?

Author: A007

**Radio Procedure**:

> - Pilots established on the eastern grass or western triangle may operate up to `A001` for low-level airwork.

Reviewer 2: To clarify, no clearance is required to conduct these low level ops? I believe at YSBK a clearance must be sought prior.

Author: Not when established on the eastern grass or western triangle no. Eg it's not uncommon at mb to do some hovering practice and training in between circuits etc. Essendon does require explicit clearance and often they will just say cleared not above 100 feet.

> - Pilots will report airborne before each circuit, or if departing additionally they will include their intentions. Aircraft are expected to remain under `A001` until you acknowledge.

Reviewer 2: So essentially, helis hover below A001 until they receive acknowledgement, then proceed for their circuit and report downwind?

Author: Helicopters don't report downwind like fixed wing traffic. Just airborne each time.

Author: ![image](https://github.com/user-attachments/assets/ff5cc989-d1a0-4036-ac4f-a62342ca6a08)

Author: I'll add this section from the ersa and clarify.

Refer to [Class D Tower Separation Standards](../../../separation-standards/classd) for more information.

---

## Runway Modes

Preferred runway modes are as follows (in order of preference):

1. Runway 35L/35R  
2. Runway 17L/17R  
3. Runways 13/31 (equal preference)

Crosswind operations are limited to **10KT**, and tailwind use is discouraged unless required.

> Runway 04/22 may only be used operationally and is **not available for circuit training**. Consideration must be given to it's limited length.

Reviewer 2: I know that the metro D's have very specific local instructions but it might be beneficial to leave this as the standard crosswind limitation that all S1s and S2s learn from the Academy. Additionally, while circuit training isn't available IRL, given our traffic levels, it might be detrimental to deny an aircraft circuits despite being the only aircraft online, because the wind favours runway 04/22. Happy to discuss, but would probably lean towards allowing circuits to improve the pilot experience, and have ATC apply normal discretion in terms of workload, etc to determine whether circuits are available or not.

---

## VFR Inbound Procedures

Aircraft should report at the standard VFR entry points:  
- **BTO** (to the west)  
- **BAW** (to the north)  
- **CARR** (to the southeast)  
- **GMH** (to the east)  
- **ACE** (to the south)  
- **SHOAL** (to the southwest)

| Entry Point | RWY 13 | RWY 17 | RWY 31 | RWY 35 |
|-------------|--------|--------|--------|--------|
| BTO         | Join final 13R | Join base 17R | Join downwind 31L | Join oblique downwind 35L |
| BAW         | Join base 13R  | Join base 17R | Join oblique downwind 31L | Join base 35L |
| CARR        | Join downwind 13R | Join downwind 17R | Join oblique base 31L | Join final 35L |
| GMH         | Join downwind 13L | Join base 17L | Join oblique base 31R | Join base 35R |
| ACE         | Join base 13L | Join oblique base 17L | Join base 31R | Join oblique downwind 35R |
| SHOAL       | Join base 13L | Join base 17L | Join base 31R | Join base 35R |

CTR Entry Altitude: `A010`.

---

## Departures into CTA

The overlying Class C airspace extends down to `A025` to the north and `A045` to the south.

- Aircraft departing into Class C must be cleared to `A050` or `RFL` (if lower) by **MB ADC**.  

> - Coordination with TCU for release is required before issuing a takeoff clearance.  

Reviewer 2: Can probably remove as we have Next coordination in the Coordination section currently

> - VFR aircraft departing into Class G must advise intentions with their "Ready" call and will be cleared accordingly.

Reviewer 2: Can probably remove, since this section details departures into CTA

Author: Western triangle and eastern grass are parts of the runway/active environment and include grass areas . Essentially they land in the unused area not used by current traffic. Similar to how at Essendon they have an eastern grass which is a portion of the aerodrome area for helicopters to train. 

So it's part of the adc controller area. I'm doing a diagram based on vatsys. I have some Mac (Moorabbin corporation) materials I am converting to vatsys based diagrams

Author: Yeah can do, this is just what was there made into more bullet point form

    **SGE**: "Cleared to leave and re-enter controlled airspace climbing to A050, Departures 129.4, SGE"

### Arrivals/Overfliers

- ML TCU will heads-up coordinate arrivals/overfliers from Class C to MB ADC.  
- **IFR arrivals** will be cleared for the coordinated approach (Instrument or Visual) prior to handoff to MB ADC, unless MB ADC nominates a restriction.  
- **VFR arrivals** will not be coordinated, and will need to receive their airways clearance from MB ADC.

!!! example
    <span class="hotline">**MDS** -> **MB ADC**</span>: "To the west, PLE, for the NDB-A"  
    <span class="hotline">**MB ADC** -> **MDS**</span>: "PLE, NDB-A"

!!! tip
    Remember that IFR aircraft are only separated from other IFR or SVFR aircraft in class D. Use other separation methods, such as holding a departure on the ground, if separation is required with these aircraft.

> ## Helicopters

Reviewer 2: Would you mind moving this section above the Coordination section, for standardisation with other pages?

### Circuits

#### Day VFR Circuits
Helicopter circuits are conducted from designated areas based on the active runway:

<figure markdown>
![Eastern Grass and Western Triangle](img/mbhelicircuitareas.png){ width="700" }
<figcaption>Eastern Grass and Western Triangle</figcaption>
</figure>

- **RWY 17/35**: Circuits are conducted from the **Eastern Grass**.
- **RWY 13/31**: Circuits are conducted from the **Western Triangle**.

> - Circuit operations are **parallel to the duty runway** and inside the fixed-wing circuit.

Reviewer 2: I'm assuming based on the example below that these areas are outside ADC's jurisdiction. Would we be able to indicate as such, so that controllers don't issue takeoff/landing clearances?

Author: Western triangle and eastern grass are parts of the runway/active environment and include grass areas . Essentially they land in the unused area not used by current traffic. Similar to how at Essendon they have an eastern grass which is a portion of the aerodrome area for helicopters to train.

So it's part of the adc controller area. I'm doing a diagram based on vatsys. I have some Mac (Moorabbin corporation) materials I am converting to vatsys based diagrams

Western triangle is extended A to the boundary fence, up to the centreline of A4, then across to the edge of B and runs parallel B to the boundary fence
Eastern grass is basically everything east of 35R
I’ll put that in the doc too tho along with the diagram

**Radio Procedure**:
- Pilots will report “AIRBORNE” before each circuit, or if departing additionally they will include their intentions. Aircraft are expected to remain under `A001` until you acknowledge.

> - Reply with simply the aircraft callsign for a leg of the circuits, or departure instructions. Hold the aircraft if they will conflict with other traffic.

Reviewer 2: ```suggestion
- Acknowledge the aircraft's callsign or provide additional departure instructions. Hold the aircraft if they will conflict with other traffic.
```

- On completion of circuit training, the pilot will nominate their landing HLS during the airborne call and report downwind before transitioning to the arrival procedure.

!!! example
    **RJB**:: "MB Gnd, RJB, At the southern apron, with information november, request circuits"
    **MB SMC**: "RJB, cleared for circuit, Eastern Grass, report airborne"      
    **RJB**: "Cleared for circuits, Eastern Grass, RJB"  
    *(Before departure)*  
    **RJB**: "Moorabbin Tower, RJB, airborne"  

>     **MB ADC**: "RJB"

Reviewer 2: I'm sure this is based off the IRL procedure, but for simplicity on VATSIM, I'd support this initial clearance call being handled by ADC. Otherwise, we will need to write coordination rules for SMC to advise ADC that an aircraft is being processed for circuits. And even just a 'cleared to operate in the circuit area, not above A010, eastern grass, report airborne', for standardisation with how it's taught in the Academy.

Author: MB changed their real life procedures this week. Before helicopters would just report to ADC their intentions for circuits.

Now they handed it off to SMC so a slip can be made. As VATPAC doesn't deal with slips its likely less of a concern, do we want to go old school helicopter rules here?

#### Night Circuits
Night operations must comply with fixed wing circuits and altitudes.

### Departures

Day VFR helicopter departures must remain at `A007`. Helicopters must nominate a aiming point or HLS on first contact with MB SMC.

1. **Aiming Points**:
   - Direct helicopters departing from the **Southern Aiming Point** to hold short of **TWY A**.
   - Direct helicopters departing from the **Northern Aiming Point** to hold short of **TWY G**.

2. **Departure Procedures**:
    - Ensure departures remain clear of duty runways and provide instructions to:
        - **Pass east of the tower**.
        - **Track**:
            - For **northbound departures**: Track north until clear of the tower.

>             - For **southbound departures**: Track south until clear of airport buildings.

Reviewer 2: Looks great, could we possibly get an example so controllers can see how this works in practice?

<figure markdown>
![Aiming Locations](img/mbhelidep.png){ width="700" }
<figcaption>Aiming Locations</figcaption>
</figure>

### Arrivals

- ML TCU will heads-up coordinate arrivals/overfliers from Class C to MB ADC.  

- **From the East**:
    - Helicopters arriving from the east should be instructed to fly a close base leg and pass over the duty threshold at **not below `A003`**.
    - For duty runways **13** or **17**, helicopters arriving for the northern or main aprons may be instructed to overfly the duty threshold at **not below `A003`** for a short right circuit.
    - For **circuits**, helicopters arriving may directly join a leg of the circuit.

- **From the West**:
    - For **southerly approaches** full stop: Instruct helicopters to remain clear of duty runways, track east of the tower, and proceed directly to the **Southern HLS** or southern aiming point.
    - For **northerly approaches** full stop: Instruct helicopters to remain clear of duty runways, track east of the tower, and proceed to the northern or main aprons.
    - For **circuits** have the aircraft fly overhead the tower and report. Instruct the aircraft to join a leg of the circuit.

!!! example

>     A helicopter, RJB, is looking to join the circuits from the west at Cerberus, with runway 17L/R active.
>     **RJB**: "MB Twr, RJB, Heli R44, Cerberus 700, inbound, with november, for circuits"
>     **MB ADC**: "RJB, MB Twr, report overhead tower"
>     **RJB**: "RJB overhead"
>     **MB ADC**: "RJB, join downwind runway 17R"
>     **RJB**: "join downwind runway, 17R"

Reviewer 2: ```suggestion
    VH-RJB, an R44 helicopter, is looking to join the circuit from the west at Cerberus, with runway 17L/R active.  
    **RJB**: "Moorabbin Tower, Helicopter RJB, Cerberus, A007, inbound, with november, for circuits"  
    **MB ADC**: "RJB, Moorabbin Tower, report overhead the tower"  
    **RJB**: "Wilco, RJB"  
    
    **RJB**: "RJB overhead the tower"
    **MB ADC**: "RJB, join downwind runway 17R"
    **RJB**: "Join downwind runway, 17R"
```

From your experience, is a 'cleared visual approach' required here, or is the absence of a level assignment sufficient to allow the pilot to descend at their discretion? I can imagine online pilots either descending early and causing conflict with other circuit traffic while overhead, or not descending at all.
